### ISSUE TITLE
Possible to support loading and introspecting via ONNX.js?

### ISSUE THREAD

**Issue by EricCousineau-TRI** [poloclub/cnn-explainer#12]
Relates #2 and #8:
Might it be possible to support loading / introspecting (a subset of) models loaded via ONNX.js?
https://github.com/Microsoft/onnxjs

Curious because we would like to visualize our own PyTorch model / networks.

I'm assuming that this may be closed as out-of-scope, which is totally fine! Just wanna put it out there ;)

**Comment by EricCousineau-TRI** [comment]
Some add'l links, on top of those provided here: https://github.com/poloclub/cnn-explainer/issues/8#issuecomment-624624004

https://medium.com/@santhoshkolloju/convert-your-pytorch-models-to-tensorflow-with-onnx-84c3bdd8d722

**Comment by xiaohk** [comment]
Hey, thanks for you suggestion!

Yeah we have considered onnx.js vs tensorflow.js when we started the project. From our experience, tensorflow.js was more straightforward at that time.

It is definitely possible to support loading models with onnx.js. In [`cnn-tf.js`](https://github.com/poloclub/cnn-explainer/blob/7a6571269cfa06382974368ce45e8f5d0d5e7894/src/utils/cnn-tf.js) we converted the loaded model into a JavaScript array of objects, and all visualization code are using this array. Therefore, after constructing this array using onnx.js all other code should work as well :)

However, it might be a little bit tricky to modify the CNN Explainer code to inspect your model (assuming not tiny-vgg 😅) out of the box, as the visualization layout is especially designed for tiny-vgg in the current version.

I will close this issue for now, let me know if you have more questions :)
